### DISCUSSION TITLE
Is it possible to use custom labels for associate form?

### DISCUSSION THREAD

**Discussion by rikiless** [filamentphp/filament#5857] [Help] [answered]
Is there way to use something like `Select::make('authorId')->getOptionLabelUsing(fn ($value): ?string => User::find($value)?->name)` also for `Tables\Actions\AssociateAction::make()`?

`getOptionLabelUsing` method is not supported for `AssociateAction`. In associate select used in relationship manager I got list of id's only and I wish to show more model attributes separated by comma for each option.

**Comment by danharrin** [comment, marked as answer]
Try `recordTitle(fn ($record) => $record->title_attribute_here)`

**Reply by rikiless** [reply]
Thanks. Working. I tried `getRecordTitle()` first and was unable to use it - it's something different.
